mne/commands/mne_browse_raw.py

```diff
@@ -89,6 +90,12 @@ def run():
     lowpass = options.lowpass
     filtorder = options.filtorder
     clipping = options.clipping
+    if isinstance(clipping, str):
+        clipping = None if clipping.lower() == 'none' else clipping
+    try:
+        clipping = float(clipping)  # allow float and convert it
+    except Exception:
```

Comment:
Can we be more specific here? We only want to proceed in case of `TypeError` and `ValueError`, right?

```suggestion
    except TypeError, ValueError:
```

Reply:
I wasn't sure what all the conditions were for `float(str)` raising an error (didn't want to bother looking), but if you're sure it's just those two I can change it

Reply:
In some testing it seems just to be `ValueError` so I'll go with that for now

Reply:
`float(None)` throws a `TypeError` :) So I guess my proposal shoud be safe

Marking as "Unresolved" so the `Commit suggestion` button becomes active again.

Reply:
Oh, you had already pushed a change to this line. Never mind.

Reply:
`None` at this point shouldn't be able to get through given our default, I guess I could change it in case we change our default back to `None` at some point?

Reply:
> `None` at this point shouldn't be able to get through given our default,

But we _do_ set `clipping = None` in certain situations in line 94 :)
see https://github.com/mne-tools/mne-python/pull/7529#pullrequestreview-398231697